src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -491,23 +498,4 @@ const EmptyConfigurationBody = styled('div')`
   padding-bottom: ${space(2)};
 `;
 
-const StyledIconCode = () => (
-  <span className="icon-code2" style={{fontWeight: 'bold', marginRight: space(1)}} />
-);
-
-const StyledIconGithub = styled(IconGithub)`
-  margin-right: ${space(1)};
-`;
-
-const StyledIconGeneric = styled(IconGeneric)`
-  margin-right: ${space(1)};
-`;
-const StyledIconDocs = styled(IconDocs)`
-  margin-right: ${space(1)};
-`;
```

Comment:
Mostly just wanted to remove these margins, since we can just use a grid for this.